.github/workflows/db-update.yml

```diff
@@ -40,6 +40,9 @@ on:
       OP_SERVICE_ACCOUNT_TOKEN:
         description: OnePassword Service Account Token
         required: true
+      OP_FEEDS_SERVICE_ACCOUNT_TOKEN:
+        description: OnePassword Service Account Token
+        required: true
```

Comment:
If this parameter is required, we need to pass it in the _db-update-dev_ and _db-update-prod_ actions

Reply:
You're right. Added